src/kernel/chainparams.cpp

```diff
@@ -267,7 +267,12 @@ class CTestNetParams : public CChainParams {
         };
 
         m_assumeutxo_data = {
-            // TODO to be specified in a future patch.
+            {
+                .height = 2'500'000,
+                .hash_serialized = AssumeutxoHash{uint256S("0x2a8fdefef3bf75fa00540ccaaaba4b5281bea94229327bdb0f7416ef1e7a645c")},
+                .nChainTx = 29324631,
```

Comment:
In 13c2066a17ba3e1dd0b81905625d1539d6e7a4f3 "chainparams: add testnet assumeutxo param at height 2_500_000"

This is `txouts`/`coins_written` (depending on which RPC you used), not the actual `nChainTx`. It should be 66484552.

Reply:
Good find, fixed.